src/applications/vaos/utils/selectors.js

```diff
@@ -64,6 +65,39 @@ export function getChosenFacilityInfo(state) {
   );
 }
 
+export function getDateTimeSelect(state, pageKey) {
```

Comment:
would be great to add unit tests for this one, especially since there's a lot going on in here.

Reply:
Good idea.  Added.